docs/security_conop/Operational_Security.rst

```diff
@@ -152,10 +152,16 @@ SIMP does not provide any direct support for contingency planning. Some of the
 mechanisms provided by SIMP might be used to support an implementation's
 contingency plan.
 
-System Backup
--------------
-
-SIMP comes with a module called ``backuppc``. This module provides a base
-configuration of the `BackupPC <http://backuppc.sourceforge.net/>`__ software
-and allows Puppet servers and clients to perform backups.
-[:ref:`CP-10 (6)`]
+System Backup and Recovery
+--------------------------
+
+SIMP does not directly support any particular backup and recovery
```

Comment:
I wish we would take this approach on a lot more things.

Reply:
@jeefberkey - We can! The goal is to have one FOSS solution for each of the NIST 800-53 sections where possible. Anything outside of that is up to the user. Additionally, if they want to disable what we have, it's fine.